### PR TITLE
fix(DB/Command): Add help options for spy command

### PR DESCRIPTION
**Pull request by Deku**
Missing options in a4d8134e388391a0e4c44cfa1756bf7561b63455

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1548117070754986000.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1548117070754986000');
+
+DELETE FROM `command` WHERE `name` IN ('spy follow', 'spy unfollow', 'spy groupfollow', 'spy groupunfollow', 'spy clear', 'spy status');
+INSERT INTO `command` (`name`,`security`,`help`) VALUES
+('spy follow',          2, 'Syntax: .spy follow [$name]\nStarts reading all the private conversations of that player.'),
```
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1548117070754986000.sql]: why are there [] around "$name" ? is it optional?

example in our db: `Syntax: .account delete $account`

Comment (Deku) [reply data/sql/updates/pending_db_world/rev_1548117070754986000.sql]: Fixed

Comment (BarbzYHOOL) [reply data/sql/updates/pending_db_world/rev_1548117070754986000.sql]: ok :+1:

File: data/sql/updates/pending_db_world/rev_1548117070754986000.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1548117070754986000');
+
+DELETE FROM `command` WHERE `name` IN ('spy follow', 'spy unfollow', 'spy groupfollow', 'spy groupunfollow', 'spy clear', 'spy status');
+INSERT INTO `command` (`name`,`security`,`help`) VALUES
+('spy follow',          2, 'Syntax: .spy follow $name\nStarts reading all the private conversations of that player.'),
```
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1548117070754986000.sql]:
```suggestion
('spy follow',          2, 'Syntax: .spy follow $name\nStart reading all the private conversations of that player.'),
```

File: data/sql/updates/pending_db_world/rev_1548117070754986000.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+('spy unfollow',        2, 'Syntax: .spy unfollow $name\nStops reading private messages from that player.'),
```
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1548117070754986000.sql]:
```suggestion
('spy unfollow',        2, 'Syntax: .spy unfollow $name\nStop reading private messages from that player.'),
```

File: data/sql/updates/pending_db_world/rev_1548117070754986000.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+('spy groupfollow',     3, 'Syntax: .spy groupfollow $name\nStarts reading all the private conversations of that player and his current party.'),
```
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1548117070754986000.sql]:
```suggestion
('spy groupfollow',     3, 'Syntax: .spy groupfollow $name\nStart reading all the private conversations from that player and from his current party.'),
```

File: data/sql/updates/pending_db_world/rev_1548117070754986000.sql [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+('spy groupunfollow',   3, 'Syntax: .spy groupunfollow $name\nStops reading private messages from that player\'s party.'),
```
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1548117070754986000.sql]:
```suggestion
('spy groupunfollow',   3, 'Syntax: .spy groupunfollow $name\nStop reading private messages from that player\'s party.'),
```

File: data/sql/updates/pending_db_world/rev_1548117070754986000.sql [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+('spy clear',           2, 'Syntax: .spy clear\nClears your current spying list, removing every player from it.'),
```
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1548117070754986000.sql]:
```suggestion
('spy clear',           2, 'Syntax: .spy clear\nClear your current spying list, removing every player from it.'),
```

**Comment by BarbzYHOOL** [comment]
Super!

**Pull request merged**